src/codegen/mod.rs

```diff
@@ -1349,20 +1349,24 @@ pub(super) fn error_msg_with_loc(ns: &Namespace, error: &str, loc: Option<Loc>)
     if let Some(loc) = loc {
         match loc {
             Loc::File(..) => {
-                let file_no = loc.file_no();
-                let curr_file = &ns.files[file_no];
-                let (line_no, offset) = curr_file.offset_to_line_column(loc.start());
-                format!(
-                    "{} in {}:{}:{}",
-                    error,
-                    curr_file.path.file_name().unwrap().to_str().unwrap(),
-                    line_no + 1,
-                    offset
-                )
+                let loc_from_file = loc_from_file(ns, loc);
+                format!("runtime_error: {error} in {loc_from_file}")
             }
             _ => error.to_string(),
         }
     } else {
         error.to_string()
     }
 }
+
+pub(super) fn loc_from_file(ns: &Namespace, loc: Loc) -> String {
+    let file_no = loc.file_no();
+    let curr_file = &ns.files[file_no];
+    let (line_no, offset) = curr_file.offset_to_line_column(loc.start());
+    format!(
+        "{}:{}:{}",
+        curr_file.path.file_name().unwrap().to_str().unwrap(),
+        line_no + 1,
+        offset
+    )
+}
```

Comment:
This function has two problems: -
 
- It only takes the start location into account, not the end. This means that user is missing same information
 - It is very similar to `ns.loc_to_string()`, however that uses the full path. 
 
 How about adding a `ns.loc_to_filename_string()` which does the same as `ns.loc_to_string()` but with only the filename and not the full path.